src/wallpaper.rs

```diff
@@ -230,14 +230,27 @@ impl Wallpaper {
 
     pub fn load_images(&mut self) {
         let mut image_queue = VecDeque::new();
+        let mut xdg_data_dirs: Vec<String> = Vec::with_capacity(5);
+        match env::var("XDG_DATA_DIRS") {
+            Ok(raw_xdg_data_dirs) => {
+                xdg_data_dirs = raw_xdg_data_dirs
+                    .split(':')
+                    .map(|s| format!("{}/backgrounds/", s))
+                    .collect();
+            }
+            Err(_) => {}
+        }
```

Comment:
`collect()` will create its own `Vec` (and, IIRC, might even use  `size_hint` for the initial capacity?).

```suggestion
        let xdg_data_dirs =  match env::var("XDG_DATA_DIRS") {
                Ok(raw_xdg_data_dirs) => {
                    xdg_data_dirs = raw_xdg_data_dirs
                        .split(':')
                        .map(|s| format!("{}/backgrounds/", s))
                        .collect();
                }
                Err(_) => Vec::new(),
        }
```

Reply:
@thefossguy Was this fixed in the latest force push?

Reply:
Yes